cgp/ea/mu_plus_lambda.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         n_processes: int = 1,
         local_search: Callable[[IndividualBase], None] = lambda combined: None,
         k_local_search: Union[int, None] = None,
+        reorder_genome: bool = True,
```

Comment:
@mschmidt87 any strong feelings about changing this default behaviour? for 1.0 i'd anyway like to use this as the default, so we might as well change it now, right?

Reply:
Should I set this default behavior to `False` for now?

Reply:
ok, do it :) we can still change it at a later point in time.